src/cpp/Makefile

```diff
@@ -1,14 +1,23 @@
 # SPDX-FileCopyrightText: Copyright (c) 2020-2025 Objectionary
 # SPDX-License-Identifier: MIT
 
-.ONESHELL:
-shell = bash
-TARGPATH = ../../targets
-
-run:
-	@mkdir $(TARGPATH)/cpp
-	for FILE in *; do \
-		if [ $$FILE != "Makefile" ]; then \
-			g++ $$FILE -o $(TARGPATH)/cpp/$${FILE%.cpp}; \
-		fi \
-	done
+CPP_FILES := $(wildcard *.cpp)
+BINARIES := $(addprefix ../../targets/cpp/, $(CPP_FILES:.cpp=.o))

```

Comment:
@MaxFunProger why keeping binaries so high? usually the say next to sources, maybe in the `target/` directory, but not in the `../../target/`